src/tables.css

```diff
@@ -0,0 +1,4 @@
+.ReactTable .-pagination {
```

Comment:
Hmmm, do you think we should create a styles/css directory for this to go into?

Reply:
Yeah you're right, we should have a directory for `styles/css` which can be a bucket of small and modular reusable bits of css styling. So really this file should be renamed as `drill-down-table.css` and `tables.css` should have styling that is shared between tables generated by both `@onaio/drill-down-table` and `@onaio/list-view` components.

Reply:
Then if we can extend that structure to and give the options of `styles/scss`, `styles/sass`, etc if needed (in the future).